client/src/style/scss/base.scss

```diff
@@ -1,3 +1,7 @@
+* {
+    -webkit-font-smoothing: antialiased;
+}
+
```

Comment:
Could you please move this underneath the imports? Imports should be kept on top for consistency.

Also since css properties are inherited there is no need to apply this to every element. Applying this to `html` or `body` should also do the trick.

Reply:
It's a bit of a deep dive that I haven't finished yet, but I also don't think we likely want to start using this rule.  It forces the browser to use antialiasing instead of native font subpixel rendering.

Reply:
Interesting! https://github.com/galaxyproject/galaxy/issues/16325#issuecomment-1611912293 suggests we could also fix the font-weights to 900 instead. Could you try if this also works @hujambo-dunia ?

Reply:
> It's a bit of a deep dive that I haven't finished yet, but I also don't think we likely want to start using this rule. It forces the browser to use antialiasing instead of native font subpixel rendering.

@dannon Good point 👍 So, after checking [Google Fonts](https://fonts.google.com/specimen/Atkinson+Hyperlegible#styles), changing to a "900" font-weight was no different than "700" because for the `Atkinson-Hyperlegible` font, there only exist Normal/400 and Bold/700 font files. However, your comment did get me digging deeper and there are several ways to solve this. In [this commit](https://github.com/galaxyproject/galaxy/pull/16515/commits/30983d50d656b70cc77aaa3a5cb0b16a5bcdc0c2), I solve it by just adding a reference to the Bold/700 version and everything appears to accept the actual Bold/700 version.

However, because previously we were actually viewing a font "adjustment"/manipulation by our web browsers to make the Normal/400 font "bold-ish" and NOT the actual Bold/700 version of this font. The actual Bold/700 looks quite a bit different than before! It's certainly distinct enough :) but maybe now too distinct? I'll add some screenshots to this PR Description to show you what I mean.

Reply:
> Interesting! [#16325 (comment)](https://github.com/galaxyproject/galaxy/issues/16325#issuecomment-1611912293) suggests we could also fix the font-weights to 900 instead. Could you try if this also works @hujambo-dunia ?

@ElectronicBlueberry Sure - see my note to Dannon above. Are you able to confirm that this `@import` reference, in [this commit](https://github.com/galaxyproject/galaxy/pull/16515/commits/30983d50d656b70cc77aaa3a5cb0b16a5bcdc0c2), is: (1) a code convention that is working (not just pulling in my local/machine installed font) and (2) a code convention we want to continue/follow?

Reply:
I cannot test this since I do not have access to a Mac. It does seem odd to me however that this import is in the masthead. I'd expect it to be in app, since it affects everything.

Reply:
@ElectronicBlueberry sure - I can check this out.

Reply:
This PR works fine for me in both Masthead.vue and onload/index.js. The latter maybe is more of an expected location for the import (also, we likely have some masthead-less pages).

I agree it is fairly distinct, but that's what I would expect from hyperlegible font anyways.

Reply:
@ElectronicBlueberry Moved to the import font-reference to the `App.vue` file in [this commit](https://github.com/galaxyproject/galaxy/pull/16515/commits/13a3c36e1081ba68564efe5de7fd171b486fdfc8). 

As @martenson mentions, there is already a more general font-reference in `onload/index.js` [shown here](https://github.com/galaxyproject/galaxy/blob/release_23.1/client/src/onload/index.js#L13) - would think we would want to bring our font-family font-references into parity at some future point to prevent confusion/duplication.

Reply:
If importing the bold font into index.js also works, this is preferable. Otherwise importing it into App is fine

Reply:
All set - upated.